src/api.v2/helpers/worker/workSubmit/submitEmbeddingWork.js

```diff
@@ -1,27 +1,21 @@
-const submitWork = require('./submitWork');
+const submitWorkForHook = require('./submitWorkForHook');
 
 const submitEmbeddingWork = async (message) => {
   const {
     experimentId, input:
-    { authJWT, config: { embeddingSettings: { method, methodSettings, useSaved } } },
+    { authJWT, config: { embeddingSettings: { method, methodSettings } } },
```

Comment:
why is `useSaved` removed?

Reply:
If we no longer need to use useSaved, we should also remove it from the API schemas and also from the worker.
Also, be careful with removing `useSaved` -- it is a parameter used by Seurat upload feature (see Alex's explanation about what `useSaved` is used for: https://github.com/hms-dbmi-cellenics/api/pull/446#discussion_r1370504471)

Reply:
when `useSaved=true` (only happens in seurat upload) we don't compute embeddings, instead we reuse the ones inside the object. So when this flag is set to true, I've changed the code so it does not precompute the embeddings neither.